server/src/services/app_import_export.service.ts

```diff
@@ -166,23 +201,22 @@ export class AppImportExportService {
       throw new BadRequestException('Invalid params for app import');
     }
 
-    let importedApp: App;
-
     const schemaUnifiedAppParams = appParams?.schemaDetails?.multiPages
       ? appParams
       : convertSinglePageSchemaToMultiPageSchema(appParams);
 
-    await dbTransactionWrap(async (manager) => {
-      importedApp = await this.createImportedAppForUser(manager, schemaUnifiedAppParams, user);
-      await this.setupImportedAppAssociations(
-        manager,
-        importedApp,
-        schemaUnifiedAppParams,
-        user,
-        externalResourceMappings
-      );
-      await this.createAdminGroupPermissions(manager, importedApp);
-    });
+    const isNormalizedAppDefinitionSchema = appParams?.schemaDetails?.normalizedAppDefinitionSchema;
```

Comment:
Can we make this decision on the basis of releasing tooljet version? Idea being when we want to decouple import flows based on version it will be easier